modules/migration-adding-cluster-to-cam.adoc

```diff
@@ -11,15 +11,15 @@ You can add a cluster to the {mtc-full} ({mtc-short}) web console.
 
 .Prerequisites
 
-If you are using Azure snapshots to copy data:
+Optional: If you are using Azure snapshots to copy data:
```

Comment:
It's not really optional. It's required if you use Azure, so you already had the restriction noted correctly. I'd revert this change.